lib/network/modules/components/nodes/shapes/CircularImage.js

```diff
@@ -58,11 +58,22 @@ class CircularImage extends CircleImageBase {
   draw(ctx, x, y, selected, hover, values) {
     this.switchImages(selected);
     this.resize();
-    this.left = x - this.width / 2;
-    this.top = y - this.height / 2;
+
+    var labelX = x,
+      labelY = y;
```

Comment:
Don't use var, please. Use const (or let if you really have to reassign the value).